src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -845,8 +859,7 @@ public override void VisitLiteralExpression(ILiteralExpression operation)
         {
             LogString(nameof(ILiteralExpression));
 
-            object value;
-            if (operation.ConstantValue.HasValue && ((value = operation.ConstantValue.Value) == null ? "null" : value.ToString()) == operation.Text)
```

Comment:
I don't understand this line. It means the raw text of the node is printed only when it's equal to calling `ToString()` on the `object` value. Shouldn't it be the other way around, that the raw text is printed when it contains something unexpected?

@mavasani As far as I can tell, you introduced this line [here](https://github.com/dotnet/roslyn/pull/11918/files?diff=unified#diff-0926b7ba2553abfab6093f42cf274213R793). Do you happen to remember why it's written like this?

Anyway, this PR doesn't change the logic, only makes it use the invariant culture. This means existing tests still pass.